src/main/java/com/subscribler/model/Package.java

```diff
@@ -0,0 +1,25 @@
+package com.subscribler.model;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Getter @Setter
+
+@Document(collection = "packages")
+public class Package {
+    @Id
+    private @NonNull String id;
+    private @NonNull String name;
+    private @NonNull String description;
+    private @NonNull int cyclePeriod; //in days
+    private @NonNull List<Item> items;
```

Comment:
I think we need to specify the item quantity in the package

Reply:
how many is ideal ?

Reply:
why tho? Isn't it best to leave it flexible?

Reply:
In each package, the merchants will have to specify the quantity of each item in the package right❓Otherwise people won't know how many they will get hahahaha

Reply:
oh I was assuming that it is just a list. so if there are multiple items of the same type it will be repeated accordingly. But I just remembered in Clarence's model it should be sth like (item, qty) instead. So should it be like a list of list (item, qty)?

Reply:
yess it should be (item, qty). Maybe we can create a new class which has the properties `item` and `quantity`. Then the list here will be list of that object

Reply:
okay. Could we call it `ItemQuantity`

Reply:
sure!